[2022-12-12 13:18:46,404] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-29T00:33:00+00:00 [queued]>
[2022-12-12 13:18:46,984] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-29T00:33:00+00:00 [queued]>
[2022-12-12 13:18:46,985] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 13:18:46,985] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-12 13:18:46,985] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 13:18:47,503] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-29 00:33:00+00:00
[2022-12-12 13:18:47,533] {standard_task_runner.py:52} INFO - Started process 2396 to run task
[2022-12-12 13:18:47,716] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-29T00:33:00+00:00', '--job-id', '869', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpdsa7cicd', '--error-file', '/tmp/tmp5dju9hqg']
[2022-12-12 13:18:47,767] {standard_task_runner.py:80} INFO - Job 869: Subtask get_last_id
[2022-12-12 13:18:50,061] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-29T00:33:00+00:00 [running]> on host b778137d8ea6
[2022-12-12 13:18:52,933] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T00:33:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-29T00:33:00+00:00
[2022-12-12 13:18:53,310] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-12 13:18:54,291] {python.py:173} INFO - Done. Returned value was: None
[2022-12-12 13:18:55,075] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221129T003300, start_date=20221212T131846, end_date=20221212T131855
[2022-12-12 13:18:55,479] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 13:18:56,471] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
